View Controllers/TweetDetailsViewController.m

```diff
@@ -0,0 +1,91 @@
+//
+//  TweetDetailsViewController.m
+//  twitter
+//
+//  Created by Sarah Wang on 6/30/21.
+//  Copyright © 2021 Emerson Malca. All rights reserved.
+//
+
+#import "TweetDetailsViewController.h"
+#import "APIManager.h"
+#import "UIImageView+AFNetworking.h"
+#import "DateTools.h"
+#import "NSDate+DateTools.h"
+
+@interface TweetDetailsViewController ()
+@property (weak, nonatomic) IBOutlet UIImageView *pfpView;
+@property (weak, nonatomic) IBOutlet UILabel *usernameLabel;
+@property (weak, nonatomic) IBOutlet UILabel *usernameHandleLabel;
+@property (weak, nonatomic) IBOutlet UILabel *timestampLabel;
+@property (weak, nonatomic) IBOutlet UILabel *tweetTextLabel;
+@property (weak, nonatomic) IBOutlet UIButton *favoriteIconView;
+@property (weak, nonatomic) IBOutlet UIButton *retweetIconView;
+@property (weak, nonatomic) IBOutlet UILabel *retweetCountLabel;
+@property (weak, nonatomic) IBOutlet UILabel *favoriteCountLabel;
+
+@end
+
+
+@implementation TweetDetailsViewController
+
+
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    [self refreshData];
+}
+
+- (void)refreshData {
```

Comment:
Same as previous review with regard to renaming to be `configureLabels` or `decorate` or something.